static/docs/user-guide/remotes/ssh.md

```diff
@@ -0,0 +1,177 @@
+# SSH
+
+We can create an SSH remote with `dvc remote add`:
+
+```dvc
+$ dvc remote add ssh-remote ssh://user@example.com/path/to/dir
+```
+
+We can modify the remote settings with `dvc remote modify`:
+
+```dvc
+$ dvc remote modify ssh-remote port 2222
+```
+
+The configuration file `.dvc/config` now should have a content like this:
+
+```ini
+['remote "ssh-remote"']
+url = ssh://user@example.com/path/to/dir
+port = 2222
+```
+
+<details>
+
+### Details: All remote options
```

Comment:
Let's not hide it. It should be similar to command reference sections - same structure for all pages, list of options, description with notes, stc

Reply:
> Let's not hide it.

Good point. I don't know why I have made it collapsed.

> It should be similar to command reference sections

It is actually a copy/paste from the command reference sections

By the way, should we keep this information duplicated (here and on the command reference), or should we remove it from the command reference?

Reply:
> By the way, should we keep this information duplicated (here and on the command reference), or should we remove it from the command reference?

Good question. I don't have a strong opinion, but it's probably better to simplify the command reference and remove duplicates.

Reply:
Done.